package.json

```diff
@@ -129,6 +129,11 @@
             {
                 "title": "LeetCode",
                 "properties": {
+                    "leetcode.showLocked": {
+                        "type": "boolean",
+                        "default": true,
```

Comment:
Suggest we set the default value to false, since I think there are more common users than the premium ones.